safeweb/http_test.go

```diff
@@ -364,3 +365,52 @@ func TestRefererPolicy(t *testing.T) {
 		})
 	}
 }
+
+func TestCSPAllowInlineStyles(t *testing.T) {
+	tests := []struct {
+		name                 string
+		cspAllowInlineStyles bool
+		wantInlineStyles     bool
+	}{
+		{
+			name:                 "CSP headers should not allow inline styles by default",
+			cspAllowInlineStyles: false,
+			wantInlineStyles:     false,
+		},
+		{
+			name:                 "CSP headers should allow inline styles when configured",
+			cspAllowInlineStyles: true,
+			wantInlineStyles:     true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
nit: since there are only 2 possible cases here, setup can be simplified
```suggestion
	for _, allow := range []bool{false, true} {
		t.Run(strconv.FormatBool(allow), func(t *testing.T) {
```